packages/synapse-react-client/test/containers/personal_access_token/CreateAccessTokenModal.test.tsx

```diff
@@ -84,7 +84,7 @@ describe('CreateAccessTokenModal tests', () => {
 
   it('does not dispatch request when there is no token name or permission', async () => {
     const { user, tokenInput, checkboxes, initialButtons } = setUp(props)
-    expect(screen.queryByTestId('ErrorBanner')).not.toBeInTheDocument()
+    expect(screen.queryByRole('alert')).not.toBeInTheDocument()
 
```

Comment:
ErrorBanner testId was removed. Find the banner by role instead, which is also a [higher priority](https://testing-library.com/docs/queries/about#priority) test query in react-testing-library